### ISSUE TITLE
Add missing casters of votes?

### ISSUE THREAD

**Issue by hancush** [datamade/nyc-council-councilmatic#34]
Two members are referenced in vote events but aren't in the API:

```
10:03:46 ERROR pupa: cannot resolve pseudo id to Person: ~{"name": "Stephen J. Fiala"}
10:03:46 ERROR pupa: cannot resolve pseudo id to Person: ~{"name": "Jose Rivera"}
```

Figure out who these gentlemen are/were and decide whether to add them to the scrape manually.

**Comment by hancush** [comment]
Fiala was a City Council member in the 1998 session: https://en.wikipedia.org/wiki/Stephen_Fiala

Rivera was a Council member (and Majority Leader) from 2001 to 2013: https://en.wikipedia.org/wiki/Joel_Rivera

**Comment by hancush** [comment]
Closed via [`742b619`](https://github.com/opencivicdata/scrapers-us-municipal/pull/171/commits/742b619fbb25e93d5ba0c7284a85d4745478492a)
